CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change history for ui-reading-room
 
-## 1.0.0 (IN PROGRESS)
+## [1.0.0] (https://github.com/folio-org/ui-eholdings/tree/v1.0.0) (2024-10-30)
```

Comment:
Petty: I think you want `[1.0.0](...)` without a space between `]` and `(` so `1.0.0` becomes the link